lib/cdo/share_filtering.rb

```diff
@@ -51,20 +48,12 @@ module FailureType
   # @param [String] program the student's program text
   # @param [String] locale a two-character ISO 639-1 language code
   def self.find_share_failure(program, locale, exceptions: false)
-    return nil unless should_filter_program(program)
-
     xml_tag_regexp = /<[^>]*>/
     program_tags_removed = program.gsub(xml_tag_regexp, "\n")
 
     find_failure(program_tags_removed, locale, exceptions: exceptions)
   end
 
-  def self.should_filter_program(program)
```

Comment:
Called by `share_filtering` above, this function restricted share filtering to ONLY programs from Play Lab apps that includes user-generated text.